components/Document/DocumentViewer.tsx

```diff
@@ -530,8 +530,8 @@ const styles = StyleSheet.create({
     width: "100%",
   },
   scroll: {
-    overflowX: "scroll",
-    overflowY: "hidden",
+    // overflowX: "scroll",
```

Comment:
Is this safe to remove?

Reply:
I believe so, but it changes the page slightly. Having these overflows causes inline comments on the first line to not show up properly